tasks/image/tag.go

```diff
@@ -2,8 +2,9 @@ package image
 
 import (
 	"fmt"
-
 	"github.com/dnephin/dobi/tasks/context"
+	"strings"
+
```

Comment:
The sorting here is wrong. `strings` should be with `fmt` as a "build-in" import